build.rs

```diff
@@ -610,11 +689,13 @@ fn yasm(file: &Path, arch: &str, out_file: &Path) -> Command {
         "x86" => ("--oformat=win32", "--machine=x86"),
         _ => panic!("unsupported arch: {}", arch),
     };
+
     let mut c = Command::new("yasm.exe");
     let _ = c.arg("-X").arg("vc")
              .arg("--dformat=cv8")
              .arg(oformat)
              .arg(machine)
+             .arg("-P").arg(&yasm_versioned_symbols_header())
```

Comment:
Instead of doing this, I think we can just change the PerlAsm translator so that it adds an `#include` to the generated asm.

Reply:
There are a couple of assembly files (`x25519-asm-$arch.S`) that aren't generated by PerlAsm, so we'd still need to do something about those, too.

Reply:
OK, thanks for reminding me of that.